src/modules/CanonicalOrders.ts

```diff
@@ -368,7 +368,7 @@ export class CanonicalOrders extends OrderSigner {
     order: CanonicalOrder,
   ): string {
     let booleanFlags = 0;
-    booleanFlags += order.isNegativeLimitFee ? 4 : 0;
+    booleanFlags += order.limitFee.isNegative() ? 4 : 0;
```

Comment:
`LimitFee` has two fields now: value and `isNegative`?

Reply:
`limitFee` is just a BigNumber which has the function `isNegative()`